src/project_types.rs

```diff
@@ -15,12 +22,32 @@ pub struct User {
     pub public_key: String,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, RustcEncodable, RustcDecodable)]
+pub struct SequenceSection {
+    pub seq_name: String,
+    pub index: u32,
+    pub num_frames: u32,
+    pub editor: Option<User>,
+    pub data: Vec<Vec<u8>>, // Row is channel, column is frame    
```

Comment:
Good point. Also, the size of data is fixed, so using a Vec seems kind of strange. Will add a separate issue for this.